feat: suzyq documentation (#253)
FyraLabs/devdocs: pages/suzyqboard/index.mdx

---

# Introduction

> The SuzyQ Board is an open-source Chromebook debug board sold by Fyra Labs. It is a versatile tool for Chromebook hacking and debugging. This board works exactly like a [Google SuzyQable](https://chromium.googlesource.com/chromiumos/third_party/hdctools/+/master/docs/ccd.md#suzyq-suzyqable). Any guides or documentation you find elsewhere on using 'SuzyQables' will work with the Fyra Labs SuzyQ Board.

Reviewer: ```suggestion
The SuzyQ Board is an open-source Chromebook debug board. It's a versatile tool for Chromebook hacking and debugging. This board works exactly like a [Google SuzyQable](https://chromium.googlesource.com/chromiumos/third_party/hdctools/+/master/docs/ccd.md#suzyq-suzyqable). Any guides or documentation you find elsewhere on using 'SuzyQables' will work with the Fyra Labs SuzyQ Board.
```

Huge shoutout to the [Chrultrabook Project](https://docs.chrultrabook.com/) for their assistance in making this product.

## FAQ

### Does My Chromebook Support Using a SuzyQ Board?

Please refer to [this list](https://www.chromium.org/chromium-os/developer-library/reference/development/developer-information-for-chrome-os-devices/), if your device supports Closed Case Debugging (CCD), it will support our SuzyQ Board.

### Where Can I Purchase a SuzyQ Board?

> You may purchase a SuzyQ Board [on the Fyra Shop](https://fyralabs.com/FDS010).

Reviewer: ```suggestion
You can buy a SuzyQ Board [from the Fyra Shop](https://fyralabs.com/FDS010).
```
maybe include the price?

---

# Introduction

The SuzyQ Board is an open-source Chromebook debug board sold by Fyra Labs. It is a versatile tool for Chromebook hacking and debugging. This board works exactly like a [Google SuzyQable](https://chromium.googlesource.com/chromiumos/third_party/hdctools/+/master/docs/ccd.md#suzyq-suzyqable). Any guides or documentation you find elsewhere on using 'SuzyQables' will work with the Fyra Labs SuzyQ Board.

> Huge shoutout to the [Chrultrabook Project](https://docs.chrultrabook.com/) for their assistance in making this product.

Reviewer: ```suggestion
Huge shoutout to the [Chrultrabook Project](https://docs.chrultrabook.com/) for their help in building this product.
```

## FAQ

### Does My Chromebook Support Using a SuzyQ Board?

Please refer to [this list](https://www.chromium.org/chromium-os/developer-library/reference/development/developer-information-for-chrome-os-devices/), if your device supports Closed Case Debugging (CCD), it will support our SuzyQ Board.

### Where Can I Purchase a SuzyQ Board?

You may purchase a SuzyQ Board [on the Fyra Shop](https://fyralabs.com/FDS010).

### Where Are the Schematics?

* [Online schematics view on PCBhub](https://pcbhub.org/projects/53)
* [GitHub repo](https://github.com/FyraLabs/suzyqboard)

### Where Can I Get Help?

If you need support with anything that may be an issue with the Fyra SuzyQ board, please join our [Discord server](https://fyralabs.com/discord) and hop into our support forum channel.

> If you have purchasing trouble, please email `contact@fyralabs.com`.

Reviewer: ```suggestion
The case is 3D printed, and snaps onto the board with its top and bottom piece.
```

Author: Wrong suggestion?

If you need support with what you're actually doing with the SuzyQ Board, and you're SURE it is not an issue with the board, your question will probably be best answered in the [Chrultrabook Forum](https://forum.chrultrabook.com/).

### What’s With the Case?

> The case is the ensure the USB-C connector does not snap off if too much force is applied.

Reviewer 2: ```suggestion
The case is to ensure the USB-C connector does not snap off if too much force is applied.
```

> The case is 3D printed, and snaps onto the board with it's top and bottom piece.

Reviewer 2: ```suggestion
The case is 3D printed, and snaps onto the board with its top and bottom piece.
```
